### PR TITLE
correct a comment in container_manager.go

### PR DESCRIPTION
**Pull request by tianshapjq**
I just found the annotations doesn't match the func and I correct it.
correct me if i'm wrong :)

### CODE REVIEW

File: pkg/kubelet/cm/container_manager.go [hunk 1]
```diff
@@ -129,11 +129,7 @@ const (
 	KubeReservedEnforcementKey    = "kube-reserved"
 )
 
-// containerManager for the kubelet is currently an injected dependency.
-// We need to parse the --qos-reserve-requests option in
-// cmd/kubelet/app/server.go and there isn't really a good place to put
-// the code.  If/When the kubelet dependency injection gets worked out,
-// maybe there will be a better place for it.
+// parsePercentage parses the string value to numeric value.
```
Comment (dixudx) [inline pkg/kubelet/cm/container_manager.go]: change to `parsePercentage parses percentage string to numeric value`.

**Review by dixudx — COMMENTED** [review]
small nit

**Comment by tianshapjq** [comment]
@dixudx @mtaufen nit done~

**Pull request merged**